website/blog/unusual-builds-w-bytes.md

```diff
@@ -0,0 +1,224 @@
+---
+slug: unusual-builds-w-bytes
+title: "Unusual Builds with Bytes"
```

Comment:
Alternative title suggestion (but this is good as is): 
* "Build with the Bytes - and then some"